mvc/views/tag-helpers/authoring.rst

```diff
@@ -1,8 +1,376 @@
+
+.. literalinclude:: authoring/sample/TagHlp/src/AuthoringTagHelpers/TagHelpers/z2AutoLinkerCopy.cs
+   :language: c#
+   :lines: 19-20
+   :dedent: 12
+   
+The code above checks to see if the content has been modified, and if it has, it gets the content from the output buffer.
+   
+7. Run the app and verify that the two links work as expected. While it might appear our auto linker tag helper is correct and complete, it has a subtle problem. If the WWW tag helper runs first, the www links will not be correct. Update the code by adding the ``Order`` overload to control the order that the tag runs in. The ``Order`` property determines the execution order relative to other tag helpers targeting the same element. The default order value is zero and instances with lower values are executed first.
+
+.. literalinclude:: authoring/sample/TagHlp/src/AuthoringTagHelpers/TagHelpers/z2AutoLinkerCopy.cs
+   :language: c#
+   :lines: 9-16
+   :emphasize-lines: 5-8
+   :dedent: 3
+
+The above code will guarantee that the WWW tag helper runs before the HTTP tag helper. Change ``Order`` to ``MaxValue`` and verify that the markup generated for the  WWW tag is incorrect.
+
+Inspecting and retrieving child content
+----------------------------------------
+
+The tag-helpers provide several properties to retrieve content.
+
+- The result of `GetChildContentAsync <https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContext.cs#L65-L79>`__ can be appended to ``output.Content``. 
+- You can inspect the result of ``GetChildContentAsync`` with ``GetContent``.
+- If you modify ``output.Content``, the TagHelper body will not be executed or rendered unless you call `GetChildContentAsync <https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContext.cs#L65-L79>`__ as in our auto-linker sample:
+
+.. literalinclude:: authoring/sample/TagHlp/src/AuthoringTagHelpers/TagHelpers/z1AutoLinkerCopy.cs
+   :language: c#
+   :lines: 8-23
+   :emphasize-lines: 6,7,11
+
+- Multiple calls to ``GetChildContentAsync`` will return the same value and will not re-execute the ``TagHelper`` body unless you pass in a false parameter indicating  not use the cached result.
+   
+Wrap up and next steps
+-----------------------
 
-.. _issue: https://github.com/aspnet/Docs/issues/232
+This tutorial was an introduction to authoring tag helpers and should not be considered a guide to best practices. For example, a real app would probably use a more elegant regular expression to replace both HTTP and WWW links in one expression. The `ASP.NET 5 MVC 6 tag helpers <https://github.com/aspnet/Mvc/tree/dev/src/Microsoft.AspNet.Mvc.TagHelpers>`__ provide the best examples of well-written tag helpers. `This GitHub repository <https://github.com/dpaquette/TagHelperSamples>`__ contains tag helper samples for working with `Bootstrap <http://getbootstrap.com/>`__. 
```

Comment:
It might be beneficial to add a link to the video that was just posed about more advanced `TagHelper` topics: https://channel9.msdn.com/Shows/Web+Camps+TV/Update-on-TagHelpers-with-Taylor-Mullen with a note to inspect comments for changes since its recording.